docs/source/llms/transformers/task.rst

```diff
@@ -0,0 +1,208 @@
+serving the model with the ``mlflow models serve`` CLI tool). If you want to load just the raw pipeline, you can
+use :py:func:`mlflow.transformers.load_model()`.
+
+Provisioned Throughput on Databricks Model Serving
+--------------------------------------------------
+
+`Provisioned Throughput <https://docs.databricks.com/en/machine-learning/foundation-models/deploy-prov-throughput-foundation-model-apis.html>`_ on Databricks Model Serving is a capability that optimizes inference performance for foundation models with performance guarantees. To serve Transformers models with provisioned throughput, specify ``llm/v1/xxx`` task type when logging the model. MLflow logs the required metadata to enable provisioned throughput on Databricks Model Serving.
+
+
+.. tip::
+
+    When logging large models, you can use ``save_pretrained=False`` to avoid creating a local copy of the model weights for saving time and disk space. Please refer to the :ref:`documentation <transformers-save-pretrained-guide>` for more details.
+
+FAQ
+---
+
+How to override the default query parameters for the OpenAI-compatible inference?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When serving the model saved with the ``llm/v1`` task type, MLflow uses the same default value as OpenAI APIs for the parameters like ``temperature`` and ``stop``. You can override them by either passing the values at inference time, or by setting different default values when logging the model.
+
+1. At inference time: You can pass the parameters as part of the input dictionary when calling the ``predict()`` method, just like how you pass the input messages.
+2. When logging the model: You can override the default values for the parameters by saving ``model_config`` parameter when logging the model.
+
+.. code-block:: python
+
+    with mlflow.start_run():
+        model_info = mlflow.transformers.log_model(
+            transformers_model=pipeline,
+            artifact_path="model",
+            task="llm/v1/chat",
+            model_config={
+                "temperature": 0.5,  # <= Set the default temperature
+                "stop": ["foo", "bar"],  # <= Set the default stop sequence
+            },
+            save_pretrained=False,
+        )
+
+
+.. attention::
+
+    The ``stop`` parameter can be used to specify the stop sequence for the ``llm/v1/chat`` and ``llm/v1/completions`` tasks. We emulate the behavior of the ``stop`` parameter in the OpenAI APIs by passing the `stopping_criteria <https://huggingface.co/docs/transformers/main_classes/text_generation#transformers.GenerationMixin.generate.stopping_criteria>`_ to the Transformers pipeline, with the token IDs of the given stop sequence. However, the behavior may not be stable because the tokenizer does not always generate the same token IDs for the same sequence in different sentences, especially for sentence-piece based tokenizers.
```

Comment:
```suggestion
    The ``stop`` parameter can be used to specify the stop sequence for the ``llm/v1/chat`` and ``llm/v1/completions`` tasks. We emulate the behavior of the ``stop`` parameter in the OpenAI APIs by passing the `stopping_criteria <https://huggingface.co/docs/transformers/main_classes/text_generation#transformers.GenerationMixin.generate.stopping_criteria>`_ to the Transformers pipeline, with the token IDs of the given stop sequence. However, the behavior may not be stable because the tokenizer does not always generate the same token IDs for the same sequence in different sentences, especially for ``sentence-piece`` based tokenizers.
```